cli/src/command/account.rs

```diff
@@ -47,7 +47,8 @@ impl CommandT for Command {
 /// Show the balance of an account
 #[derive(StructOpt, Clone)]
 pub struct ShowBalance {
-    /// SS58 address
+    /// SS58 address that can be given directly or
+    /// obtained given a local account name.
```

Comment:
This is displayed to the user so we should tell them what to input.
```suggestion
    /// SS58 address or name of a local account.
```
Please also adjust the `value_name`. It should be `address_or_account`.

Reply:
> we should tell them what to input.

That's what it does now: it tells users that the param represents an SS58 address that can be given directly or obtained given a local account name.

I am fine with your edition though.